llvm/lib/Target/PowerPC/PPCAsmPrinter.cpp

```diff
@@ -1292,8 +1291,9 @@ void PPCAsmPrinter::emitInstruction(const MachineInstr *MI) {
 
     unsigned Op = MI->getOpcode();
 
-    // Change the opcode to load address for tocdata
-    TmpInst.setOpcode(Op == PPC::ADDItocL8 ? PPC::ADDI8 : PPC::LA);
+    // Change the opcode to load address for toc data.
```

Comment:
Another nit: maybe `toc-data` instead of `toc data` for consistency with other comments.